packages/prisma-client/migrations-cli/README.md

```diff
@@ -0,0 +1,50 @@
+## CLI Reference
+
+### `--dev`
+
+This flag can be added to any command. It informs the CLI that it’s used in a development environment, which slightly changes its behaviour:
+
+- automatically loads variables from `.env` files,
+- uses a lockfile,
```

Comment:
which lockfile? for what?

Reply:
>  for what?

To make sure you don't run two migration processes in parallel. I probably should say that in the doc.

> which lockfile?

Do you mean where it's located?

Reply:
yeah, I had no idea, need to explain that this tool generates an own lockfile to do XYZ, I thought you mean yarnlock or something